src/java17/groovy/eu/xenit/gradle/enterprise/conventions/extensions/dockerimagelabels/SpringBootDockerImageLabelsApplier.groovy

```diff
@@ -0,0 +1,32 @@
+package eu.xenit.gradle.enterprise.conventions.extensions.dockerimagelabels
+
+import org.gradle.api.Project
+import org.gradle.api.Task
+import org.gradle.api.provider.Provider
+
+class SpringBootDockerImageLabelsApplier implements BuildContextInformationApplier {
+
+    void accept(Project project, Provider<BuildContextInformation> buildContextInformationProvider) {
+        project.getPlugins().withId("org.springframework.boot") { plugin ->
+            def pluginLoader = new PluginClassLoader(plugin)
+
+            Class<Task> bootBuildImageClass = pluginLoader.loadClass("org.springframework.boot.gradle.tasks.bundling.BootBuildImage")
+
+            project.getTasks()
+                    .withType(bootBuildImageClass)
+                    .configureEach {
+                        environment.putAll(buildContextInformationProvider.map {
+                            buildContextInformation ->
```

Comment:
`GradleDockerPluginImageLabelApplier` also sets `.version` and `.title` - are those already applied by the spring-boot plugin ?

Reply:
Yes, those are automatically set by the spring boot plugin